backend/src/main/java/se/chalmers/dat265/group1/storage/PostgresSQLConnector.java

```diff
@@ -29,10 +30,7 @@ private void establishConnection() throws SQLException {
         this.connectionProperties.put("user", USERNAME);
         this.connectionProperties.put("password", PWD);
 
-
-        this.conn = DriverManager.getConnection(
-                this.CONNECTION_URL,
-                connectionProperties);
+        this.conn = DriverManager.getConnection(this.CONNECTION_URL, connectionProperties);
 
         System.out.println("Connected to database");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)